### PR TITLE
[CBRD-25393] Add new test case for Oracle-Style CREATE VIEW Statement with NULL Values

### PR DESCRIPTION
**Pull request by zionyun**
refer to http://jira.cubrid.org/browse/CBRD-25393

### CODE REVIEW

**Review by ssihil — APPROVED** [review]

**Review by vimkim — APPROVED** [review]

File: sql/_35_fig_cake/cbrd_25393/cases/cbrd_25393.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+DROP VIEW IF EXISTS a_view;
+
+CREATE TABLE a_tbl (
+    id INT NOT NULL,
+    phone VARCHAR(10)
+);
+INSERT INTO a_tbl VALUES(1,'111-1111'), (2,'222-2222'), (3, '333-3333'), (4, NULL), (5, NULL);
+
+SELECT * FROM a_tbl;
+
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25393/cases/cbrd_25393.sql:14]: SELECT * FROM a_tbl ORDER BY id;

File: sql/_35_fig_cake/cbrd_25393/cases/cbrd_25393.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "ID", "PHONE", "EXTRA"
+) AS
+SELECT
+    id,
+    phone,
+    NULL AS extra
+FROM a_tbl;
+
+SELECT * FROM a_view;
+
```
Comment (kwonhoil) [inline sql/_35_fig_cake/cbrd_25393/cases/cbrd_25393.sql:31]: SELECT * FROM a_view ORDER BY ID;

**Review by kwonhoil — APPROVED** [review]

File: sql/_35_fig_cake/cbrd_25393/cases/cbrd_25393.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    id,
+    phone,
+    NULL AS extra
+FROM a_tbl;
+
+SELECT * FROM a_view ORDER BY id;
+
+DROP TABLE IF EXISTS a_tbl;
+DROP VIEW IF EXISTS a_view;
+
```
Comment (bagus-kim) [inline sql/_35_fig_cake/cbrd_25393/cases/cbrd_25393.sql]: --Add case
--사이트에서 발생한 이슈로, 담당의 test-case를 등록했으면 합니다.

create or replace view "v_test"("id", "name") as
select 1 as id, sysdate name from dual
union all
select null as id, null as name from dual;

Comment (zionyun) [reply sql/_35_fig_cake/cbrd_25393/cases/cbrd_25393.sql]: 네 등록했습니다.

**Review by bagus-kim — APPROVED** [review]

**Review by tw-kang — APPROVED** [review]
